trace-utils/src/send_data.rs

```diff
@@ -725,16 +737,27 @@ mod tests {
     async fn request_msgpack() {
         let server = MockServer::start_async().await;
 
+        let header_tags = HEADER_TAGS;
         let mock = server
             .mock_async(|when, then| {
                 when.method(POST)
+                    .header(HEADER_DD_TRACE_COUNT, "1")
                     .header("Content-type", "application/msgpack")
+                    .header("datadog-meta-lang", header_tags.lang)
+                    .header(
+                        "datadog-meta-lang-interpreter",
+                        header_tags.lang_interpreter,
+                    )
+                    .header("datadog-meta-lang-version", header_tags.lang_version)
+                    .header("datadog-meta-lang-vendor", header_tags.lang_vendor)
+                    .header("datadog-meta-tracer-version", header_tags.tracer_version)
+                    .header("datadog-container-id", header_tags.container_id)
```

Comment:
@hoolioh The to HashMap [implementation](https://github.com/DataDog/libdatadog/blob/main/trace-utils/src/tracer_header_tags.rs#L40) for `TracerHeadertags` always ignores `client_computed_stats` and `client_computed_top_level` tags. I assume that is intentional?

Reply:
I assume that's because PHP don't support it. cf [this](https://github.com/DataDog/dd-trace-php/blob/92560faa353e7d5412a0b0517d13f4e9a0506d0c/ext/auto_flush.c#L66). @bwoebi can confirm.

That said we should open a ticket for it not to forget. It's gonna be important when we will compute stats on the client side.

Reply:
APMSP-1167 for the follow up

Reply:
Yeah, no need for it yet.